api/events/schema.py

```diff
@@ -64,6 +65,30 @@ def mutate(self, info, **kwargs):
         return CancelEvent(event=event)
 
 
+class EndEvent(graphene.Mutation):
+    """
+    Mutation to end an event
+    Returns event payload on ending the event
+    """
+    class Arguments:
+        calendar_id = graphene.String(required=True)
+        event_id = graphene.String(required=True)
+        start_time = graphene.String(required=True)
+        end_time = graphene.String(required=True)
```

Comment:
This can be `actual_end_time` and then the next one can be `expected_end_time` depending on what they each of them means.

Reply:
Based on the discussion we had, a refactor task can be added to rename the `meeting_end_time` to `check_out_time` or `actual_end_time`